composer.json

```diff
@@ -15,7 +15,18 @@
         "optimize-autoloader": true
     },
     "require": {
-        "magento/module-quote": ">=101.1.3",
```

Comment:
Curious why we're removing this dependency?

Reply:
That's a great question! I'm meeting with Claire to discuss if we can. When I searched online, I couldn't find a reference to this plugin in packagist, magento slack community, or POGS (_plain old google search_). It looks like it was added two years ago, and it may just in included as part of magento 2's source code now.

Reply:
To update this for posterity: We decided to add a `composer.dev.json` for local development, which will get used within the makefile as `make composer-dev`. More [here](https://klaviyo.slack.com/archives/C046U293FDM/p1675960628020699). and in PR #230.